### PR TITLE
Remove all manager name references.

### LINKED ISSUE #23316: Remove manager name from vulnerability document
## Description

This issue aims to remove the manager name information from the vulnerability document.

This is crucial to avoid indicating node information in the indexer, since this is not valuable to the user and can cause certain inconsistencies or inconsistencies between documents.

### PR DESCRIPTION
**Pull request by Dwordcito**
|Related issue|
|---|
|Close #23316 |

## Description
This PR aims to delete all information related to the manager name, because this information can cause a cosmetic incongruences, because a vulnerability could be detected by multiple nodes when an agent move from one node to another node.

### CODE REVIEW

**Review by GabrielEValenzuela — APPROVED** [review]
Great job!

The ode is correct and the tests were successful! 🚀

File: src/wazuh_modules/vulnerability_scanner/src/policyManager/policyManager.hpp [hunk 1]
```diff
@@ -116,10 +116,7 @@ class PolicyManager final : public Singleton<PolicyManager>
             newPolicy["indexer"]["ssl"]["key"] = "";
         }
         newPolicy["indexer"]["name"] =
-            STATES_VD_INDEX_NAME_PREFIX +
-            (newPolicy.at("vulnerability-detection").contains("clusterName")
-                 ? newPolicy.at("vulnerability-detection").at("clusterName").get_ref<const std::string&>()
-                 : "default");
+            STATES_VD_INDEX_NAME_PREFIX + newPolicy.at("clusterName").get_ref<const std::string&>();
```
Comment (sebasfalcone) [inline src/wazuh_modules/vulnerability_scanner/src/policyManager/policyManager.hpp:119]: This should be lower-case formatted due to restrictions on the indexer
```json
{
    "index":
    {
        "_index": "wazuh-states-vulnerabilities-Thinkpad-Sebas",
        "_id": "001_ff5f53a5ed3d2659c0f0b5de579951d332ca47b3_CVE-2024-22667",
        "status": 400,
        "error":
        {
            "type": "invalid_index_name_exception",
            "reason": "Invalid index name [wazuh-states-vulnerabilities-Thinkpad-Sebas], must be lowercase",
            "index": "wazuh-states-vulnerabilities-Thinkpad-Sebas",
            "index_uuid": "_na_"
        }
    }
}
```

```suggestion
            STATES_VD_INDEX_NAME_PREFIX + Utils::toLowerCase(newPolicy.at("clusterName").get_ref<const std::string&>());
```

Comment (sebasfalcone) [reply src/wazuh_modules/vulnerability_scanner/src/policyManager/policyManager.hpp:119]: After the fix, vulnerabilities are indexed
image

**Review by sebasfalcone — CHANGES_REQUESTED** [review]
In a non-cluster environment, the hostname is used as name:
image

```
root@Thinkpad-Sebas:/var/ossec/bin# hostname
Thinkpad-Sebas
```

File: src/wazuh_modules/vulnerability_scanner/src/policyManager/policyManager.hpp (hunk 1 above)
Comment (Dwordcito) [reply src/wazuh_modules/vulnerability_scanner/src/policyManager/policyManager.hpp:119]: good catch!

**Review by sebasfalcone — DISMISSED** [review]
LGTM!

- Same result as in 4.8.0
image

- Cluster name is the same as hostname in non-cluster environment
image


- Index name is lower-case
image

**Review by GabrielEValenzuela — APPROVED** [review]
LGTM.

Great work!

**Pull request merged**